README.md

```diff
@@ -94,6 +94,13 @@ The following peripherals are supported in `avr-hal-generic`:
 
 ### HAL Status
 The chip-HAL crates currently support the following peripherals:
+* [`atmega2560-hal`](./chips/atmega2560-hal)
+  - [x] Spinning Delay
+  - [x] `PORTA`, `PORTB`, `PORTC`, `PORTD`, `PORTE`, `PORTF`, `PORTG`, `PORTH`, `PORTJ`, `PORTK`, `PORTL` as digital IO
+  - [x] `USART0`, `USART1`, `USART2`, `USART3` for serial communication
+  - [x] I2C using `TWI`
+  - [x] SPI
+  - [x] ADC
```

Comment:
As discussed, please remove the ones which aren't yet implemented.

Reply:
Done.